.github/workflows/update-docs.yml

```
@@ -15,7 +15,84 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Prerequisites
+        run: |
+          sudo apt-get install doxygen
+          sudo apt-get install nodejs
+          npm install moxygen
+
+      - name: Check out python sdk
+        uses: actions/checkout@master
+        with:
+          repository: LucidtechAI/las-sdk-python
+          path: python-sdk
+
       - name: Set up Python 3.8
         uses: actions/setup-python@v2
         with:
           python-version: 3.8
+
+      - name: Python SDK
+        run: |
+          cd python-sdk
+          pip install -r requirements.docs.txt
+          pip install tox
+          make docs
+          mv docs/las.md ../reference/python-sdk/latest.md
+
+      - name: Check out java sdk
+        uses: actions/checkout@master
+        with:
+          repository: LucidtechAI/las-sdk-java
+          path: java-sdk
+
+      - name: Java SDK
+        run: |
+          cd java-sdk
+          doxygen
+          npx moxygen xml
+          mv api.md ../reference/java/latest.md
+
+      - name: Check out .NET sdk
+        uses: actions/checkout@master
+        with:
+          repository: LucidtechAI/las-sdk-net
+          path: net-sdk
+
+      - name: .NET SDK
+        run: |
+          cd net-sdk
+          doxygen documentation.conf
+          npx moxygen xml
+          mv api.md ../reference/dotnet/latest.md
+
+      - name: Check out java-script sdk
+        uses: actions/checkout@master
+        with:
+          repository: LucidtechAI/las-sdk-js
+          path: js-sdk
+
+      - name: JavaScript SDK
+        run: |
+          cd js-sdk
+          npm install && npm run upgrade-lucidtech
+          make docs
```

There is an npm script that should be used instead here, the old makefile command will probably be removed and is outdated (oops).

```suggestion
          npm run docs
```